addons/mail/static/src/components/chat_window/chat_window.xml

```diff
@@ -21,7 +21,7 @@
                     t-ref="header"
                 />
                 <t t-if="!chatWindow.isFolded">
-                    <t t-if="chatWindow.thread">
+                    <t t-if="chatWindow.threadViewer">
```

Comment:
Strange to have to change these conditions in component.

Reply:
I thought it was more logical to "t-if" if a threadviewer exists before using <ThreadViewer> component, no ?

Reply:
In general I would say yes, but in chat window, thread has a particular meaning, it is the differentiation between a "new message" chat window, and an actual chat window with someone.

To know if the "new message" stuff has to be shown, it must be an "else" of "thread" and not of "thread viewer" (which can be removed for other reasons).

Reply:
Makes sense indeed